filecheck/cli.ts

```diff
@@ -29,7 +29,7 @@ program
     }
     return runChecker(allFilePaths, options).catch((error) => {
       console.error(error);
-      process.exit(1);
+      throw error;
```

Comment:
Manually changed this to match the other CLI because of the `n/no-process-exit` rule

Reply:
LGTM